docs/01 - build.md

```diff
@@ -109,17 +129,55 @@ If you want to reuse an existing Kubernetes config, such as your [`EKS`](https:/
 
-
-
-### Setup a Kubernetes development environment
-To run `KubeBlocks`, you needs `Docker` and a `Kubernetes` 1.24.1+ cluster for development.
-
-#### Docker environment
-1. Install [Docker](https://docs.docker.com/install/)
-   > For Linux, you'll have to configure docker to run without `sudo` for the KubeBlocks build scripts to work. Follow the instructions to [manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
-2. Create your [Docker Hub account](https://hub.docker.com/signup) if you don't already have one.
-
-#### Kubernetes environment
-- Kubernetes cluster
-  You can use cloud Kubernetes service, such as [`EKS`](https://aws.amazon.com/eks/) [`GKE`](https://cloud.google.com/kubernetes-engine) [`AKS`](https://azure.microsoft.com/en-us/products/kubernetes-service/), or use local Kubernetes cluster, such as [`Minikube`](https://minikube.sigs.k8s.io/docs/) [`k3d`](https://k3d.io/stable/).
-- For development purposes, you will also want to follow the optional steps to install [Helm 3.x](https://helm.sh/docs/intro/install/).
+## Deploying released version of KubeBlocks
+Here we present an example of deploying released version of `KubeBlocks` locally and utilizing it to create an `apeloud-mysql` cluster. 
+
+#### Setup Kubernetes environment
+- Setup local cluster(Minikube, k3d), e.g., `k3d cluster create mycluster`
+#### Install `KubeBlocks`
+- Install [`kbcli`](https://kubeblocks.io/docs/release-0.8/user_docs/installation/install-with-kbcli/install-kbcli)
+- Install [`KubeBlocks`](https://kubeblocks.io/docs/release-0.8/user_docs/installation/install-with-kbcli/install-kubeblocks-with-kbcli): `kbcli kubeblocks install`. This will install `KubeBlocks` and start running its manager.
+
+#### Install `apecloud-mysql` addon 
+Databases can be added as addons in `KubeBlocks`. To test with `apecloud-mysql`, you need to install and enable corresponding addon. You can install addons through [Helm](https://kubeblocks.io/docs/release-0.8/developer_docs/integration/how-to-add-an-add-on). Here is a simpler example using [`kbcli`](https://kubeblocks.io/docs/preview/user_docs/overview/supported-addons#list-addons):
+
+```shell
+# make sure there is an index
+kbcli addon index list
+# search supported addon
+kbcli addon search apecloud-mysql  
+# install addon
+kbcli addon install apecloud-mysql --index kubeblocks --version [YOUR_VERSION]
+# enable addon
+kbcli addon enable apecloud-mysql
+# list the addons again to check whether it is enabled.
+kbcli addon list
+```
+#### Test with `apecloud-mysql`
+Here, we present a simple example of creating an `apecloud-mysql` cluster.
+- [Create `apecloud-mysql` cluster](https://kubeblocks.io/docs/preview/user_docs/kubeblocks-for-mysql/cluster-management/create-and-connect-a-mysql-cluster) 
```

Comment:
The links provided in the above documentation refer to version 0.8, while the current document links to the preview version. It is recommended to keep them consistent.

Reply:
It's suggested developper to work on the latest version of KB.

Reply:
`kbcli cluster create mysql mycluster` works on both kbcli 0.8.x and kbcli 0.9, and user doc in KubeBlocks 0.8 suggest using `kbcli cluster create mysql mycluster` .  I find that some addons such as mysql/redis support this command as shortcut command while others must set `--cluster-definition`.

To better align with the user documentation, I'll make the change to use `MySQL` as an example instead of `apecloud-mysql`.